source/build/Windows/TestClientApp.vcxproj.ejs

```diff
@@ -86,7 +86,7 @@
       <PrecompiledHeader>Use</PrecompiledHeader>
       <WarningLevel>Level4</WarningLevel>
       <SDLCheck>true</SDLCheck>
-      <PreprocessorDefinitions>_CONSOLE;WIN32;<%- clientDefines %>%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_CONSOLE;WIN32;<% if(azureSdk){ %> <%- azureDefines %> <% } %><%- clientDefines %>%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
NIT: Use azureDefines in the way that clientDefines is used, and remove the need for azureSdk.